tests/test-timeout.js

```diff
@@ -43,6 +43,19 @@ if (process.env.TRAVIS === 'true') {
     })
   })
 
+  tape('should set connect to false', function(t) {
```

Comment:
Note Travis skips the tests in this file. I verified the tests passed on my local machine (a Mac). Would appreciate testing in other environments, especially Windows - I'm not sure if Node has the same socket API on Windows machines.